plugins/scm.js

```diff
@@ -107,7 +120,7 @@ const UPDATE_COMMIT_STATUS = Joi.object().keys({
     scmUri,
     token,
     sha,
-    buildStatus,
+    buildStatus: Joi.string().valid(...githubStatus, ...gitlabStatus),
```

Comment:
is this supposed to be a mandatory field?

Reply:
Yes, I think there is no scenario where we can update a commit without its status.

Reply:
@jithine do you want to add anything here?

Reply:
then it should have a  .required() I believe

Reply:
Added the required() validation. Thank you.